### PR TITLE
Fixed css issues

### PR DESCRIPTION
**Pull request by roberthouse54**
#### What are the relevant tickets?

There is not an issue for this.

#### What's this PR do?

This PR fixes an issue where the navbar was disappearing at small browser widths. I added a responsive style sheet to fix this. Any other responsive styles should be addressed in this new scss file. I also fixed a few small style tweaks on the user search page. Button style, a few variable color changes, etc.

### CODE REVIEW

**Comment by aliceriot** [comment]
do you think this closes #753 ?

**Comment by aliceriot** [comment]
looks good though! the navbar still appears with a narrow screen.

File: static/scss/responsive_styles.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
+@media screen and (max-width: 1024px) {
+
```
Comment (aliceriot) [inline static/scss/responsive_styles.scss]: can you remove this line?

File: static/scss/responsive_styles.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  .mdl-layout__header-row {
+    height: 64px;
+    padding: 0 40px 0 80px;
+  }
+
+  .mdl-layout__tab {
+    padding: 0 24px 0 24px;
+  }
+
```
Comment (aliceriot) [inline static/scss/responsive_styles.scss]: and this one?

**Comment by aliceriot** [comment]
two little formatting things, otherwise looks good!

feel free to merge once you've deleted those lines. :+1:

**Comment by aliceriot** [comment]
oh looks like there's a merge conflict with #1004

**Comment by roberthouse54** [comment]
@aliceriot I would say this does not close #753

File: static/scss/responsive_styles.scss (hunk 2 above)
Comment (roberthouse54) [inline static/scss/responsive_styles.scss]: OK, I fixed these formatting issues.

**Comment by noisecapella** [comment]
I'm going to take this over to rebase, fix the review comments and merge (cc @roberthouse54)

**Pull request merged**